pkg/cli/store.go

```diff
@@ -35,7 +35,7 @@ func init() {
 
 	// Set of all flags used across GUAC clis and subcommands. Use consistent
 	// names for config file.
-	set.String("nats-addr", "nats://127.0.0.1:4222", "address to connect to NATs Server")
+	set.String("pubsub-addr", "nats://127.0.0.1:4222", "address to connect to a pubsub (default is NATS)")
```

Comment:
Instead of "address" maybe "gocloud connection string"?

Reply:
fixed